stories/components/button/__stories__/components/ButtonSizes.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Button from 'intergalactic/button';
```

Comment:
Let's use `@semcore` here for consistency

Reply:
fixed